DataAccessLayer/Repositories/Implementation/WordTranslationRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DataAccessLayer.DataBaseModels;
+using DataAccessLayer.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Repositories.Implementation
+{
+    public class WordTranslationRepository : ICrudRepository<WordTranslation>
+    {
+        private LanguageSkillsDBContext _db;
+        public WordTranslationRepository(LanguageSkillsDBContext context)
+        {
+            this._db = context;
+        }
+
+        public IEnumerable<WordTranslation> GetAll()
+        {
+            return _db.WordTranslations;
+        }
+
+        public WordTranslation Get(int id)
+        {
+            return _db.WordTranslations.Find(id);
+        }
+
+        public void Create(WordTranslation wordTranslation)
+        {
+            _db.WordTranslations.Add(wordTranslation);
+        }
+
+        public void Update(WordTranslation wordTranslation)
+        {
+            _db.Entry(wordTranslation).State = EntityState.Modified;
```

Comment:
It may not work.
Please debug and make sure DB record is indeed updated

Reply:
This code has been successfully tested for workability